bin/package-requires-update

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env node
+
+var exec = require( 'child-process-promise' ).exec;
+var packageJson = require( '../' + process.argv[2] );
+var semver = require( 'semver' );
+
+if ( packageJson.private ) {
+	process.exitCode = 0;
+}
+
+if ( /[^A-Za-z0-9_\/:=-]/.test( packageJson.name ) ) {
+	console.log( packageJson.name, 'Package name cannot contain spaces and/or invalid characters' );
+	console.log( '\n' );
+	process.exitCode = 1;
+} else {
+	exec( 'npm view ' + packageJson.name + ' version' )
+		.then( function( result ) {
+			if ( semver.gt( packageJson.version, result.stdout ) ) {
+				process.exitCode = 0;
+			} else {
+				console.log( packageJson.name, 'Package must be versioned' );
```

Comment:
I meant that the package version must be bumped.  I have cleaned up the messaging in:  https://github.com/Automattic/wp-calypso/commit/e8bdacacf3569372b362198e85850c830b9a7aca